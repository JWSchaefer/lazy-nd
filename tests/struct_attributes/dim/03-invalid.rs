use lazy_nd::lazy_nd;

#[lazy_nd(dim = false)]
struct Test {}

fn main() {}
