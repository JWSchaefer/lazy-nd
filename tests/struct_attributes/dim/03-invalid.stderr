error: expected identifier or integer literal
 --> tests/struct_attributes/dim/03-invalid.rs:3:17
  |
3 | #[lazy_nd(dim = false)]
  |                 ^^^^^
